src/super_gradients/recipes/quantization_params/default_quantization_params.yaml

```diff
@@ -0,0 +1,21 @@
+modify_recipe_for_qat:
```

Comment:
See summary regarding extracting theses recommended practices parameters as a pre_launch_callback.
This should look pretty much the same actually, but just at the top of the config (i.e the inheriting recipe from the original).